enhancements/compound-bundle-constraints.md

```diff
@@ -238,7 +242,7 @@ properties:
 - type: olm.constraint
   value:
     message: Cannot be required for Baz because...
```

Comment:
Also, can you do me a favor and update this EP to covert the `message` field in all examples to `failureMessage` instead? We recently updated that part in the code to honor the suggest from Joe in my generic constraint EP.

Reply:
Yep, I'll add this.